docs/DafnyRef/UserGuide.md

```diff
@@ -401,14 +396,14 @@ This list is not exhaustive but can definitely be useful to provide the next ste
 
 ### 24.8.2. Verification debugging when verification is slow {#sec-verification-debugging-slow}
 
-In this section, we describe techniques to apply in the case when verification is slower than expected, does not terminate, or timeouts.
+In this section, we describe techniques to apply in the case when verification is slower than expected, does not terminate, or times out.
 
 #### 24.8.2.1. `assume false;` {#sec-assume-false}
 
 Assuming `false` is an empirical way to short-circuit the verifier and usually stop verification at a given point[^explainer-assume-false], and since the final compilation steps do not accept this command, it is safe to use it during development.
 Another similar command, `assert false;`, would also short-circuit the verifier, but it would still make the verifier try to prove `false`, which can also lead to timeouts.
 
-[^explainer-assume-false]: `assume false` tells the Dafny verifier "Assume everything is true from this point of the program". The reason is that, 'false' proves anything. For example, `false ==> A` is always true because it is equivalent to `!false || A`, which reduces to `true || A`, which reduces to `true`.
+[^explainer-assume-false]: `assume false` tells the `dafny` verifier "Assume everything is true from this point of the program". The reason is that, 'false' proves anything. For example, `false ==> A` is always true because it is equivalent to `!false || A`, which reduces to `true || A`, which reduces to `true`.
```

Comment:
Why do you prefer `dafny` over Dafny here?

Reply:
You are right that it could be either -- I was thinking that the verifier was part of the tool, but alternately the Dafny language system has a verifier though the language itself does not